OData/src/System.Web.OData/OData/ODataSwaggerUtilities.cs

```diff
@@ -0,0 +1,633 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System.Diagnostics.CodeAnalysis;
+using System.Diagnostics.Contracts;
+using System.Linq;
+using Microsoft.OData.Edm;
+using Newtonsoft.Json.Linq;
+
+namespace System.Web.OData
+{
+    /// <summary>
+    ///  Utility methods used to convert the Swagger model.
+    /// </summary>
+    [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "o", Justification = "Utils is spelled correctly.")]
+    public static class ODataSwaggerUtilities
```

Comment:
Is it possible to make this override-able, so customer can overwrite specific function in this class and no need to override ODataSwaggerConverter. For example,  the service has some entitysets which are not allowed to be posted, so the swagger service document should not tell the customer that the entityset can be posted, then if this class is override-able, the service developer just need override CreateSwaggerPathForEntitySet.